samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/4.2-SelfServiceRegistrationPasswordOptional.feature

```diff
@@ -0,0 +1,62 @@
+﻿Feature: 4.2 Self Service Registration Password Optional
+
+User registeration scenario for password optional feature
+
+Background: 
+	Given a Profile Enrollment policy defined assigning new users to the Everyone Group and by collecting "First Name", "Last Name", and "Email", is allowed and assigned to a SPA, WEB APP or MOBILE application
+	And "Required before access is granted" is selected for Email Verification under Profile Enrollment in Security > Profile Enrollment
+	And configured Authenticators are Email (required) and Password (optional)
+	And a user named "Mary"
+	And Mary does not have an account in the org
+
+  @ignore
```

Comment:
Is there a reason that this is ignored?

Reply:
yes, ignored those because email passcode comes in twice so sometimes the test fails if it picks the first code and not the second. I believe we prob had this conversation before about that limitation.